### ISSUE TITLE
docs/integration: a single-page intro to tech CUE integrates with

### ISSUE THREAD

**Issue by jpluscplusm** [cue-lang/docs-and-content#80]
Turn the existing content-less URL into a high-level, tickbox, "yes, CUE works with X" for folks wanting to sense check if they can make use of CUE.

This page:

- pull in any content from its subpages, as needed
  - don't replicate any "XYZ is planned" language
    - make sure that an issue/roadmap-entry/etc exists, as appropriate, tracking the aim
- include language-specific pointers to other (possibly new/moved) docs
  - e.g. so that a Go user can see a compact list of CUE capabilities
     - with links pointing to a more complete search/tag page
- don't include any technical content or examples on the page, unless they're *very* short.
  - this should primarily be links to other docs
- remove/migrate all sub-pages, in line with the points below
- language:
  - don't use the word ""integration"", as it implies there's a tangible "thing", which makes it harder to differentiate "integrations" (which aren't tangible) from "adapters" (which are/will-be tangible, usable things)
    - use the phrase "integrates with" on this page to point to specific capabilities (e.g. JSON/YAML)

Subpages:

Adapt and reword subpages' content into new, newcomer-friendly concept guides.

Migrate most of the "JSON" and "YAML" content to concept/{json,yaml} as appropriate. And/or docs/tour (when it's showing off a feature) or docs/introduction (when we want to make sure the reader knows that a core capability exists)

**Comment by jpluscplusm** [comment]
https://cue-lang/cuelang.org/cl/1183402

**Comment by jpluscplusm** [comment]
Current status of CL (https://cuelang.org/cl/1183402 (has comments) superseded by https://cuelang.org/cl/1192707): blocked on #72 & #75.

**Comment by jpluscplusm** [comment]
Previous CL: https://cuelang.org/cl/1183402 (has comments)
Current CL: https://cuelang.org/cl/1192707 (supersedes previous)

**Comment by jpluscplusm** [comment]
This issue currently reduces to:

- re-home `/docs/integration/go/` content appropriately so that anything on that page that's both unique and valid is still available; so that we can
- remove all left-hand nav elements from `/docs/integration/`; but
- leave behind aliases redirecting `/docs/integration/X/` to `/docs/concept/how-cue-works-with-X/`; and then
- rename `/docs/integration/` to `/docs/integrations/` with an appropriate alias.
